code/ingest/src/main/java/org/jesterj/ingest/processors/FetchUrl.java

```diff
@@ -48,6 +48,8 @@
   private long throttleMs;
   private String errorField;
   private String httpStatusField;
+  private int ConnectionTimeOut = 5000; //defaulting to 5000 ms
```

Comment:
Just noticed.... field naming convention in java is normally to have lower case first letter. (Initial capital is reserved for class names, etc)